org.mozilla.Firefox.json

```diff
@@ -165,6 +165,11 @@
         },
         {
           "type": "file",
+          "url": "https://github.com/endlessm/eos-shell-content/raw/master/icons/bundle/64x64/apps/eos-app-org.mozilla.Firefox.png",
```

Comment:
The URL should be added to the .in manifest as well. It's not being used right now but we should start using it.